README.md

```diff
@@ -4,17 +4,8 @@ Plugin for `Yazi` to preview all supported files via `Bat`.
 
 ## Preview
 
-Showing .csv files
+![preview](https://github.com/user-attachments/assets/67fe6e52-95c6-4768-a378-732e9c11cb84)
```

Comment:
The preview image is hosted outside of the repo. Please put the preview image into the repo